concepts/ORM/Querylanguage.md

```diff
@@ -8,16 +8,11 @@ The syntax supported by Sails' model methods is called Waterline Query Language.
 The criteria objects are formed using one of four types of object keys. These are the top level
 keys used in a query object. It is loosely based on the criteria used in MongoDB with a few slight variations.
 
-Queries can be built using either a `where` key to specify attributes, which will allow you to also use query options such as `limit` and `skip` or, if `where` is excluded, the entire object will be treated as a `where` criteria.
+Queries can be built using either a `where` key to specify attributes, or excluding it.
 
-```javascript
-
-var peopleNamedMary = await Model.find({
-  name: 'mary'
-});
-
-// OR
+Using the `where` key allows you to also use [criteria clauses](#criteria_clauses), such as `limit`, `skip`, and `sort`.
```

Comment:
When the docs are recompiled for the Sails website, all the headings are automatically turned into links, so there's no need to manually set one up here -- before I merge this in, would you change this link to point at https://sailsjs.com/documentation/concepts/models-and-orm/query-language#?criteria-clauses, then remove that link below and just leave it as a regular header?

Reply:
@rachaelshaw this should be resolved now thx